socialhome/streams/app/components/Stream.vue

```diff
@@ -35,5 +34,20 @@
     import "streams/app/components/StreamElement.vue"
     import store from "streams/app/stores/streamStore"
 
-    export default Vue.component("stream", {store})
+    export default Vue.component("stream", {
+        store,
+        data() {
+            return {
+                masonryOptions: {
+                    "item-selector":       ".grid-item",
+                    "column-width":        ".grid-sizer",
+                    "gutter":              ".gutter-sizer",
+                    "percent-position":    true,
+                    "stamp":               ".stamped",
+                    "transition-duration": "0s",
+                    "stagger":             0
```

Comment:
We don't have an ESlint config yet, but soon will once I add one - and dangling commas will be there since they make diffs better and look pythonic. So good idea to start getting used to adding commas on the last multi-line lines! :)